### PR TITLE
Install docs experiments (DOC-7865)

### PR DESCRIPTION
**Pull request by paperclypse**
This is a draft/experiment. Please don't merge.

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

**Review by rhetoric101 — DISMISSED** [review]
@paperclypse, I'm still not quite sure how to do reviews in CodeStream. I inserted some separate codemarks, but I'm not sure if there's a way to link the codemarks to this PR.


Anyway, let me know if you have any questions.

File: src/content/docs/apm/agents/java-agent/getting-started/monitor-your-java-app.mdx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  - /docs/agents/java-agent/frameworks/grails-installation-java
+  - /node/2306
+  - /docs/agents/java-agent/installation/java-agent-manual-installation
+  - /docs/agents/java-agent/installation/java-agent-self-installer
+  - /docs/apm/agents/java-agent/installation/install-java-agent
+---
+
+import javaApmExampleImage from 'images/java-apm-example-dashboards.png'
+
+Want to start monitoring your Java application with New Relic right away? Use this guide to find the best path to start sending your application data to New Relic.
```
Comment (rhetoric101) [inline src/content/docs/apm/agents/java-agent/getting-started/monitor-your-java-app.mdx:48]: @paperclypse, here's a comment I'm testing right in the DIFF view of CodeStream!

File: src/content/docs/apm/agents/java-agent/getting-started/monitor-your-java-app.mdx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  * [Heroku installation](/docs/apm/agents/java-agent/heroku/java-agent-heroku)
+  * [Java agent configuration](/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file)
+
+  </Collapser>
+
+</CollapserGroup>
+
+## What's next? [#next]
+
+Already sending data to New Relic from your Java application? Here are some next steps:
```
Comment (paperclypse) [inline src/content/docs/apm/agents/java-agent/getting-started/monitor-your-java-app.mdx:137]: This is a test.

File: src/content/docs/apm/agents/java-agent/getting-started/monitor-your-java-app.mdx (hunk 1 above)
Comment (paperclypse) [reply src/content/docs/apm/agents/java-agent/getting-started/monitor-your-java-app.mdx:48]: I've finally had time to get to these! Thanks for all your comments, @rhetoric101!

**Pull request merged**